pkg/operation/botanist/component/gardeneraccess/access.go

```diff
@@ -110,6 +111,15 @@ func (g *gardener) Deploy(ctx context.Context) error {
 }
 
 func (g *gardener) Destroy(ctx context.Context) error {
+	var shootAccessSecretNames = []string{v1beta1constants.SecretNameGardener, v1beta1constants.SecretNameGardenerInternal}
+
+	for _, v := range shootAccessSecretNames {
+		secret := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: v, Namespace: g.namespace}}
+		if err := client.IgnoreNotFound(g.client.Delete(ctx, secret)); err != nil {
```

Comment:
This works, but typically we do it like this:
```suggestion
		if err := g.client.Delete(ctx, secret); client.IgnoreNotFound(err) != nil {
```